app/src/main/java/mollie/tictactoe/activities/BoardActivity.java

```diff
@@ -0,0 +1,84 @@
+package mollie.tictactoe.activities;
+
+import android.app.AlertDialog;
+import android.app.Dialog;
+import android.content.DialogInterface;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.view.View;
+
+import mollie.tictactoe.GameHelper;
+import mollie.tictactoe.R;
+import mollie.tictactoe.StateManager;
+import mollie.tictactoe.ui.BoardView;
+import mollie.tictactoe.ui.CellButton;
+import mollie.tictactoe.ui.UIBoardManager;
+
+public class BoardActivity extends AppCompatActivity {
+
+    private GameHelper mGameHelper = new GameHelper();
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
```

Comment:
I would look up for what `super` is doing. `@Override` and `super` usually go together.